tests/snapshots.rs

```diff
@@ -605,12 +605,9 @@ fn convert_spv_all() {
         true,
         Targets::HLSL | Targets::WGSL | Targets::METAL,
     );
-    convert_spv(
-        "empty-global-name",
-        true,
-        Targets::HLSL | Targets::WGSL | Targets::METAL,
-    );
```

Comment:
Was removing this intended?

Reply:
Yeah, it seems to be duplicated 👀 (unless this duplication is part of the test, ofc. - but it doesn't seem to be)